<?xml version="1.0" encoding="UTF-8"?>
    <ui version="4.0">
     <class>MainWindow</class>
     <widget class="QMainWindow" name="MainWindow">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1300</width>
        <height>700</height>
       </rect>
      </property>
      <property name="windowTitle">
       <string>Gesti√≥n de Productos</string>
      </property>
      <widget class="QWidget" name="centralwidget">
       <layout class="QHBoxLayout" name="horizontalLayout_main">
        <item>
         <!-- Panel Izquierdo: Lista de Productos -->
         <widget class="QGroupBox" name="groupBox_lista">
          <property name="title">
           <string>Inventario de Productos</string>
          </property>
          <property name="minimumWidth">
           <number>650</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_lista">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_buscar">
             <item>
              <widget class="QLineEdit" name="lineEdit_buscar">
               <property name="placeholderText">
                <string>Buscar por nombre o c√≥digo...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_buscar">
               <property name="text">
                <string>üîç Buscar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_filtros">
             <item>
              <widget class="QLabel" name="label_filtro_categoria">
               <property name="text">
                <string>Categor√≠a:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_filtro_categoria">
               <item>
                <property name="text">
                 <string>-- Todas las categor√≠as --</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_filtro_precio">
               <property name="text">
                <string>Precio:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_precio_min">
               <property name="maximumWidth">
                <number>100</number>
               </property>
               <property name="placeholderText">
                <string>M√≠n</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_guion">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_precio_max">
               <property name="maximumWidth">
                <number>100</number>
               </property>
               <property name="placeholderText">
                <string>M√°x</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_aplicar_filtros">
               <property name="text">
                <string>Filtrar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_limpiar_filtros">
               <property name="text">
                <string>Limpiar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget_productos">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <column>
              <property name="text">
               <string>C√≥digo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Nombre</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Valor Adquisici√≥n</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Valor Venta</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Ganancia</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Cantidad</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Categor√≠a</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_botones_tabla">
             <item>
              <widget class="QLabel" name="label_total">
               <property name="text">
                <string>Total: 0 productos</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_refrescar">
               <property name="text">
                <string>üîÑ Refrescar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_nuevo">
               <property name="text">
                <string>‚ûï Nuevo Producto</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <!-- Panel Derecho: Detalles del Producto -->
         <widget class="QGroupBox" name="groupBox_detalles">
          <property name="title">
           <string>Detalles del Producto</string>
          </property>
          <property name="minimumWidth">
           <number>450</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_detalles">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <layout class="QVBoxLayout" name="verticalLayout_scroll">
               <item>
                <widget class="QGroupBox" name="groupBox_info_basica">
                 <property name="title">
                  <string>Informaci√≥n B√°sica</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_basicos">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_codigo">
                    <property name="text">
                     <string>C√≥digo *:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_codigo">
                    <property name="placeholderText">
                     <string>C√≥digo √∫nico del producto</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_nombre">
                    <property name="text">
                     <string>Nombre *:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_nombre">
                    <property name="placeholderText">
                     <string>Nombre del producto</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_descripcion">
                    <property name="text">
                     <string>Descripci√≥n:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QTextEdit" name="textEdit_descripcion">
                    <property name="maximumHeight">
                     <number>80</number>
                    </property>
                    <property name="placeholderText">
                     <string>Descripci√≥n detallada del producto...</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_categoria">
                    <property name="text">
                     <string>Categor√≠a *:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QComboBox" name="comboBox_categoria">
                    <item>
                     <property name="text">
                      <string>-- Seleccione una categor√≠a --</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_cantidad">
                    <property name="text">
                     <string>Cantidad en Stock:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QSpinBox" name="spinBox_cantidad">
                    <property name="minimum">
                     <number>0</number>
                    </property>
                    <property name="maximum">
                     <number>999999</number>
                    </property>
                    <property name="value">
                     <number>0</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_precios">
                 <property name="title">
                  <string>Precios y Costos</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_precios">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_valor_adquisicion">
                    <property name="text">
                     <string>Valor de Adquisici√≥n *:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_valor_adquisicion">
                    <property name="placeholderText">
                     <string>0.00</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_valor_venta">
                    <property name="text">
                     <string>Valor de Venta:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_valor_venta">
                    <property name="placeholderText">
                     <string>Se calcular√° autom√°ticamente</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="2">
                   <widget class="QPushButton" name="pushButton_calcular_venta">
                    <property name="text">
                     <string>üßÆ Calcular Precio de Venta</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0" colspan="2">
                   <widget class="QLabel" name="label_info_calculo">
                    <property name="text">
                     <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;&lt;span style="font-size:9pt; font-style:italic;"&gt;El precio de venta se calcula seg√∫n IVA y Utilidad de la categor√≠a&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_resumen">
                 <property name="title">
                  <string>Resumen Financiero</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_resumen">
                  <item>
                   <widget class="QLabel" name="label_resumen_adquisicion">
                    <property name="text">
                     <string>üì• Costo de Adquisici√≥n: $0.00</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_resumen_venta">
                    <property name="text">
                     <string>üí∞ Precio de Venta: $0.00</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_resumen_ganancia">
                    <property name="text">
                     <string>üìà Ganancia Estimada: $0.00 (0.00%)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_resumen_iva">
                    <property name="text">
                     <string>üßæ IVA Incluido: $0.00</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_modo">
             <property name="text">
              <string>&lt;html&gt;&lt;body&gt;&lt;p align="center"&gt;&lt;span style="font-weight:600;"&gt;Modo: Visualizaci√≥n&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_acciones">
             <item>
              <widget class="QPushButton" name="pushButton_guardar">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumHeight">
                <number>35</number>
               </property>
               <property name="text">
                <string>üíæ Guardar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_editar">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumHeight">
                <number>35</number>
               </property>
               <property name="text">
                <string>‚úèÔ∏è Editar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_eliminar">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumHeight">
                <number>35</number>
               </property>
               <property name="text">
                <string>üóëÔ∏è Eliminar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_cancelar">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumHeight">
                <number>35</number>
               </property>
               <property name="text">
                <string>‚ùå Cancelar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QStatusBar" name="statusbar"/>
     </widget>
     <resources/>
     <connections/>
    </ui>